index.js

```diff
@@ -11,35 +11,40 @@ function TaserReporter(baseReporterDecorator, config) {
     // This is not a full list of events that can be listened for, only the set we currently care about
     this.onRunStart = onRunStart.bind(this);
     this.onBrowserStart = onBrowserStart.bind(this);
-    this.onBrowserError = onBrowserError.bind(this);
+    this.onBrowserError = onBrowserStart.bind(this);
     this.onSpecComplete = onSpecComplete.bind(this);
     this.onRunComplete = onRunComplete.bind(this);
 
     function onRunStart() {
         browsers = {};
         errors = {};
+
     }
-    
-    function onBrowserStart(browser) {
-        browsers[browser.id] = {
-            browser: {
-                name: browser.name,
-                fullName: browser.fullName
-            },
-            testResults: {
-                passed: [],
-                ignored: [],
-                failed: []
-            },
-            errors: errors[browser.id] || []
-        };
-    }
-    
-    function onBrowserError(browser, error) {
-        errors[browser.id] = errors[browser.id] || [];
-        errors[browser.id].push(error);
+
+    function onBrowserStart(browser, error) {
```

Comment:
If onBrowserStart ever hands in a second parameter that is a string and not actually an onBrowserError event, this code will be tough to debug. They should be separate functions